iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -31,8 +31,8 @@
   <div id="inspector" class="no-fact-selected">
     <div id="inspector-head">
       <div class="title">
-        <h1 class="footnote-mode-off">Fact Properties</h1>
-        <h1 class="footnote-mode-only">Footnote Properties</h1>
+        <h1 class="footnote-mode-off" data-i18n="inspector.factProperties">Fact Properties</h1>
+        <h1 class="footnote-mode-only" data-i18n="inspector.footNoteProperties">Footnote Properties</h1>
```

Comment:
This is obviously fine, but I would have expected `footnoteProperties`.

Reply:
Done in c975ed1